homeassistant/components/camera/netatmo.py

```diff
@@ -76,7 +76,7 @@ def camera_image(self):
                     self._localurl), timeout=10)
             else:
                 response = requests.get('{0}/live/snapshot_720.jpg'.format(
-                    self._vpnurl), timeout=10)
+                    self._vpnurl), timeout=10, verify=False)
```

Comment:
I agree with @pvizeli. Disabling verification of SSL is huge. It means that you render SSL useless. If this fix requires adding that option, then we should push out a config change in a minor release. It's defaulting to the same behavior so it's not going to force people to update their config if it already works today.

Disabling SSL verification is a big band-aid. I wonder why it is needed to begin with, doesn't Debian Jessie have the right root certificates or is netatmo using self signed certificates?